### ISSUE TITLE
Manage case concierge process

### ISSUE THREAD

**Issue by lippytak** [codeforamerica/clean#413]
Stub. Everyone on the team will do some but I'll make sure it's easy and cases don't fall through the cracks.

Get all Info Sharing cases into a spreadsheet (pre-data storage emails)

**Comment by lippytak** [comment]
Some considerations:
- For right now should we contact clients directly or only HSA?
- Are we comfortable using our personal contact info (cell, email) to contact clients? If not what's easiest way to setup a call/text/email communication with clients? Will clients understand who we are?
- Worth setting up some sort of support/ticket tracking like Zendesk or just use a spreadsheet?
- What is easiest way to map application data to HSA case data? Clean App ID?

**Comment by daguar** [comment]
> For right now should we contact clients directly or only HSA?

Both. We'll learn more.

> Are we comfortable using our personal contact info (cell, email) to contact clients? If not what's easiest way to setup a call/text/email communication with clients? Will clients understand who we are?

I'd vote for a single phone # that we all share — maybe a quick Twilio-based thing? I have to imagine this exists already, but if not, I can set something up quick.

> Worth setting up some sort of support/ticket tracking like Zendesk or just use a spreadsheet?

I defer to you, but a spreadsheet probably works for now — status + notes field. But I can see the case for using Zendesk, especially in terms of multiple interactions where a single "notes" field doesn't suffice. We could also be kinda nuts and create a private repo and use Issues for tracking (I kinda like that idea.) Just make a call and I'll do whatever.

> What is easiest way to map application data to HSA case data? Clean App ID?

Yeah, let's just construct a fake one right now and put it in both the Carissa spreadsheet and our concierging one as a cross-walk. Eventually we can redo this.

**Comment by lippytak** [comment]
see https://devcenter.heroku.com/articles/pgstudio

and think about GH + Waffle workflow

**Comment by daguar** [comment]
Def prefer that to a console view :)

**Comment by lippytak** [comment]
also need to get crosswalk (app ID) into clean-cases spreadsheet

**Comment by lippytak** [comment]
Also this could work: https://groove.groovehq.com/knowledge_base/topics/custom-profile-app

**Comment by lippytak** [comment]
Does our job end when the client is approved or when they successfully use their EBT card for the first time?

**Comment by daguar** [comment]
@lippytak — In our funnel, we don't have any measure of EBT card use. If we care about that as the outcome (I could be persuaded either way) I'd suggest we measure comprehensively for all cases as the final stage.

**Comment by lippytak** [comment]
@daguar apparently the info exists in CW so we could do it. I'll discuss with Carissa today.
![screen shot 2015-03-13 at 11 26 14 am](https://cloud.githubusercontent.com/assets/2533112/6644743/c746a8d0-c973-11e4-938c-2529861df954.png)

**Comment by lippytak** [comment]
**Your mission**
These are your clients: https://docs.google.com/spreadsheets/d/10V35ZFq26RbCPzV_M-zfKH--UHkLyqmcrB_qLILErKo/edit#gid=1562188399 (see Concierge sheet / Owner col). Do whatever you can to make them super happy. If that means dropping off an EBT card at their work, do it.

**Get clients approved**
- First, understand exactly why cases are pending/denied and what you/the client can do about it by calling SF HSA at (415) 558-1001.
- For missed interviews
  - See if HSA can try again (I don't know if they can/will ever do that)
  - If you reach client, encourage them to call HSA and schedule an interview.
  - If they're retisant, consider calling for them, waiting on hold, and initiating a 3-way call when the EW picks up
- For missing verification docs
  - Make sure you understand exactly what they need and ask me/Liliana/Carissa/EW about details before you talk to the client. Remember you can self-verify income, rent, utilities, child care.
  - If client has email they can send things to food@sfgov.org with their case info (case number and/or name + DOB)
  - If client has MMS they can send things to us and we'll submit for them
  - If client prefers they can fax to (415) 355-2300 or mail to CalFresh, P.O. Box 7988, San Francisco, CA 94120

**Things to keep in mind**
-  Always be super nice to eligibility workers. Ask hard questions but understand what they can't do and remember they are on our team but stuck in a frustrating system. If they go out of their way to help remember their name and we'll send them a thank you card later or something.
- Since HSA is reporting case info to us, we should NOT have to call for basic info. If you need more info to do effective client follow up, note that and we'll ask Carissa to start collecting it.
- For non-English cases, just work on the HSA side for now (other ideas?)
- You can use your personal phone or the burner app http://www.burnerapp.com/ (I'll be using personal for now)

**Going forward I will**
- Update concierge spreadsheet on Mondays
- Make sure we have a crosswalk soon (app ID in cases tab) so we don't need to call HSA to get the case number (issue: https://github.com/codeforamerica/clean/issues/423)

**Comment by lippytak** [comment]
@alanjosephwilliams and @daguar take a look at ^ and LMK if it's missing anything important. Otherwise we'll adapt as we go.

**Comment by daguar** [comment]
> this works well: "if you have a case number then I can jot that down and we can use that next time if I have to call"
- @lippytak

**Comment by daguar** [comment]
Just want to say awesome job to all concierging yesterday, and thanks @lippytak for managing the process with aplomb.

**Comment by lippytak** [comment]
@gabbyjones I finally write up 'case follow up' instructions: https://github.com/codeforamerica/clean/wiki/How-to-do-case-follow-up

I would **love** if you could try this out for a couple cases when you get the chance! Feel free to start or let's discuss soon.
